### PR TITLE
NXP-25846: add Drive-like exporter plugin

### PR DESCRIPTION
**Pull request by dmetzler**
Update packaging to make plugin configurable thru a `nuxeo.conf` property

### CODE REVIEW

File: marketplace/src/main/resources/install/templates/fs-exporter/nuxeo.defaults [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Edit this file to customize your configuration ##
+fs-exporter.target=.
+
+# Set list of templates to include, comma separated values.
+nuxeo.template.includes=common
+
+nuxeo.fs-exporter.plugin.class=org.nuxeo.io.fsexporter.DefaultExporterPlugin
```
Comment (egiuly-nuxeo) [inline marketplace/src/main/resources/install/templates/fs-exporter/nuxeo.defaults]: why do you put the class in the template?

(btw missing blank line at end of file)

**Review by egiuly-nuxeo — APPROVED** [review]

File: marketplace/src/main/resources/install/templates/fs-exporter/nuxeo.defaults (hunk 1 above)
Comment (dmetzler) [reply marketplace/src/main/resources/install/templates/fs-exporter/nuxeo.defaults]: For the default value of the contribution. That way, the OPS just have to update the `nuxeo.conf` file to update the plugin behavior.

**Review by mcedica — APPROVED** [review]

File: marketplace/src/main/resources/install/templates/fs-exporter/nuxeo.defaults (hunk 1 above)
Comment (efge) [reply marketplace/src/main/resources/install/templates/fs-exporter/nuxeo.defaults]: It's better to have the default in the code than in the config. Putting classes in config is something we should avoid.

Comment (dmetzler) [reply marketplace/src/main/resources/install/templates/fs-exporter/nuxeo.defaults]: The thing is that the configuration is in an extension point. It has to be configurable by the ops team thru `nuxeo.conf`. We could put an indirection name -> class but for the current need, it's really not worth it. We will probably have to rewrite that whole module at some point.

Comment (dmetzler) [reply marketplace/src/main/resources/install/templates/fs-exporter/nuxeo.defaults]: Moved as a comment and changed template in `nxftl` to better handle default case.

**Pull request merged**